pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -439,10 +449,13 @@ lib.makeOverridable (
           if [ -z "''${dontStrip-}" ]; then
             installFlagsArray+=("INSTALL_MOD_STRIP=1")
           fi
+
+          installFlagsArray+=("INSTALL_MOD_PATH=''${outputs[modules]}")
```

Comment:
This could just be `INSTALL_MOD_PATH=${placeholder "modules"}` in the flags, right? Am I misunderstanding?